playbooks/ansible-core-ci/pre.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
So, we need to create a new job in project-config, that we will parent too in aws jobs. That will generate the token on the file-system. that will minimized the change to leak the secret.